scripts/fetch-external-resources

```diff
@@ -30,6 +31,12 @@ RESOURCES = [
     nil,
     'content/doc/pipeline/steps'
   ],
+  [
+    'https://repo.jenkins-ci.org/api/search/versions?g=org.jenkins-ci.main&a=jenkins-core&repos=releases&v=?.*.1',
+    'content/_data/_generated/lts_baselines.yml',
```

Comment:
This is the awesome power of YAML being a superset of JSON. Artifactory only has JSON API responses, and I'm lazy, and this works, so 🤷‍♂️